apps/prairielearn/elements/pl-threejs/pl-threejs.py

```diff
@@ -623,35 +613,29 @@ def get_orientation(element, name_orientation, name_format):
                 return np.roll((qx * qy * qz).elements, -1).tolist()
             else:
                 raise ValueError()
-        except Exception:
+        except Exception as err:
             raise Exception(
-                'attribute "{:s}" with format "{:s}" must be a set of roll, pitch, yaw angles in degrees with format "[roll, pitch, yaw]": {:s}'.format(
-                    name_orientation, name_format, s
-                )
-            )
+                f'attribute "{name_orientation:s}" with format "{name_format:s}" must be a set of roll, pitch, yaw angles in degrees with format "[roll, pitch, yaw]": {s:s}'
+            ) from err
```

Comment:
There seems to be overlap between this and https://github.com/PrairieLearn/PrairieLearn/pull/11058. Are there multiple rules that are enforcing the same thing?

Reply:
No, this is just a merge conflict caused by cherry-picking. Removed in 9fce7e0eb. I had expected this sort of stuff to happen since so many lint rules target the same set of lines.